### PR TITLE
Fix #6 and test the inclusion problem of both fast and rt Issue #7.

### LINKED ISSUE #6: Double template declaration and incompatibility between fast and rt
Templates print and distance are defined both in https://github.com/stack-of-tasks/eiquadprog/blob/b08d57dd401b64e20ddb184a2eda9131cc34e006/include/eiquadprog/eiquadprog-fast.hxx#L27 and https://github.com/stack-of-tasks/eiquadprog/blob/b08d57dd401b64e20ddb184a2eda9131cc34e006/include/eiquadprog/eiquadprog-rt.hpp#L209

They should be put in one hxx file included in both hxx file.
A PR will follow with a test to check that it is working

### PR DESCRIPTION
**Pull request by olivier-stasse**
Add a unittest showing the problem with classical use of the header as they are right now.

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -22,9 +22,16 @@ SET(TESTS
...
 
 FOREACH(test ${TESTS})
   ADD_UNIT_TEST(${test} ${test}.cpp)
   TARGET_LINK_LIBRARIES(${test} ${PROJECT_NAME} ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
 ENDFOREACH(test ${TESTS})
+
+ADD_LIBRARY(testab SHARED TestA.cpp TestB.cpp)
+TARGET_INCLUDE_DIRECTORIES(testab SYSTEM PRIVATE ${EIGEN3_INCLUDE_DIR})
+
+#ADD_UNIT_TEST(test-integration test-integration.cpp )
```
Comment (nim65s) [inline tests/CMakeLists.txt:36]: Why is this commented ?

File: tests/eiquadprog-both.cpp [hunk 2]
```diff
@@ -0,0 +1,165 @@
+//
+// Copyright (c) 2019 CNRS
```
Comment (nim65s) [inline tests/eiquadprog-both.cpp:2]: I think we are in 2020

File: tests/CMakeLists.txt (hunk 1 above)
Comment (olivier-stasse) [reply tests/CMakeLists.txt]: Fixed in commit ca274884

File: tests/eiquadprog-both.cpp (hunk 2 above)
Comment (olivier-stasse) [reply tests/eiquadprog-both.cpp]: Fixed in commit 23c9dfb

**Comment by olivier-stasse** [comment]
The PR passed https://gepgitlab.laas.fr/ostasse/eiquadprog/pipelines/9172

**Comment by nim65s** [comment]
Thanks for this !

@gabrielebndn : do you have any remarks ?

**Comment by olivier-stasse** [comment]
Thanks @andreadelprete !

**Pull request merged**